packages/perseus/src/widgets/phet-sim.tsx

```diff
@@ -215,6 +215,6 @@ export default {
     displayName: "PhET Simulation",
     widget: PhetSim,
     // Let's not expose it to all content creators yet
-    hidden: true,
+    hidden: false,
```

Comment:
Wooooohooo! 🎉

Reply:
At risk of raining on everyone's parade, we still have a few minor design/a11y tickets to push through before totally releasing! But because we have the feature branch, so the code won't hit prod until everything's done, I think okay to un-hide! And release coming soon 👀

Reply:
Okay, re-hiding so we can double check it works on TestEverything before we fully launch!